kentikapi/internal/httputil/httputil.go

```diff
@@ -106,9 +86,12 @@ func defaultHTTPClient() *http.Client {
 // makeRetryPolicy creates customized retry policy.
 // Its implementation is based on retryablehttp.ErrorPropagatedRetryPolicy.
 // Retry policy function returns true if the request should be retried.
-func makeRetryPolicy(statusCodes []int, methods []string) retryablehttp.CheckRetry {
-	statusCodesSet := makeIntSet(statusCodes)
-	methodsSet := makeStringSet(methods)
+func makeRetryPolicy() retryablehttp.CheckRetry {
+	statusCodesSet := makeIntSet([]int{429, 500, 502, 503, 504})
+	methodsSet := makeStringSet([]string{
```

Comment:
How about removing methodsSet and `if !isRequestRetryable(resp.Request, methodsSet) {` block of code? It is not needed now.

Reply:
Done